.ci.yaml

```diff
@@ -4316,3 +4316,8 @@ targets:
         ["devicelab","android","windows"]
       task_name: windows_chrome_dev_mode
     scheduler: luci
+
+  - name: google_test
+    bringup: true
+    presubmit: false
+    scheduler: google_internal
```

Comment:
Tags are needed here to correctly identify the test owners.
```
properties:
      tags: >
          ["framework","hostonly","shard"]
```

Reply:
Thanks! Added.

Reply:
Is it easy to add tags? This should be marked `external` and ignored by the flaky test bot

Reply:
Related issue: https://github.com/flutter/flutter/issues/91913.

Do we want to keep the builder as `bringup: true`, or will remove flake flag later? If the former case, an easy workaround for now if adding the test in the ignore list: https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/request_handlers/deflake_flaky_builders.dart#L43

Otherwise, we need to escalate the above referenced bug.

Reply:
@renyou you'll also need to add an entry in `TESTOWNERS`

Reply:
Added, is this right?